lib/pr_checker.js

```diff
@@ -199,9 +198,9 @@ class PRChecker {
     }
   }
 
-  hasFullOrLiteCI(ciMap) {
+  hasFullCI(ciMap) {
     const cis = [...ciMap.keys()];
-    return cis.find(ci => isFullCI(ci) || isLiteCI(ci));
+    return cis.find(isFullCI);
```

Comment:
Nit: could this be `cis.some()`?

Reply:
Both work and both make sense, I don't think there's an upside to changing it